ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs

```diff
@@ -527,6 +527,32 @@ public void Create(String filePath)
         }
     }
 
+    public class CFDataBarNegative : IXLExample
+    {
+        public void Create(String filePath)
+        {
+            var workbook = new XLWorkbook();
+            var ws = workbook.AddWorksheet("Sheet1");
+
+            ws.FirstCell().SetValue(-1)
+                .CellBelow().SetValue(1)
+                .CellBelow().SetValue(2)
+                .CellBelow().SetValue(3);
+
+            byte[] bytes = new byte[16];
+            BitConverter.GetBytes(1).CopyTo(bytes, 0);
+            var guid = new Guid(bytes);
+
+            var conditionalFormat = ws.RangeUsed().AddConditionalFormat();
+            conditionalFormat.Name = string.Concat("{", guid.ToString(), "}");
```

Comment:
Referring to other comments, why are you setting the `Name` here?

Reply:
Because the name is a guid and autogenerate. I can't reproduce the same file in resource because when créate the example asignate a newguid, except I set manually the name

Reply:
Sorry, but the variable Name is need to identify the id of conditionaformat between ConditionalformattingRuleExtension and WorksheetExtension because the XML tag are in different position of file.
It's posible make the SET internal

Reply:
Ok, I'll have a look at it. Thanks for your contributions so far.